chart/templates/jobs/create-user-job.yaml

```diff
@@ -48,6 +48,9 @@ metadata:
     {{- $annotations | toYaml | nindent 4 }}
   {{- end }}
 spec:
+  {{- if not (kindIs "invalid" .Values.createUserJob.ttlSecondsAfterFinished) }}
```

Comment:
Checking specifically for `not nil` because `0` is a valid value for this field and it would evaluate to false normally.